### PR TITLE
Tamara

### PR DESCRIPTION
**Pull request by Aramat16ukr**
I'm done with my work

### CODE REVIEW

File: src/test/java/Group4/Tamara_T2.java [hunk 1]
```diff
@@ -5,6 +5,7 @@
     public static void main(String[] args) {
         int[] nums={1,5,9,23,13};
         System.out.println(solution(nums));
+
```
Comment (BadamkhandB) [inline src/test/java/Group4/Tamara_T2.java:8]: Looks Great

**Review by BadamkhandB — APPROVED** [review]
Great

**Review by KarinaFrei — APPROVED** [review]
Great work!

**Review by ZaierAouani — APPROVED** [review]
Great!

**Review by natebell510 — APPROVED** [review]
A

**Review by msebbata — APPROVED** [review]
good

**Pull request closed without merging**